enterprise/internal/batches/sources/bitbucketserver.go

```diff
@@ -252,3 +252,23 @@ func (s BitbucketServerSource) CreateComment(ctx context.Context, c *Changeset,
 
 	return s.client.CreatePullRequestComment(ctx, pr, text)
 }
+
+// MergeChangeset merges a Changeset on the code host, if in a mergeable state.
+// If squash is true, and the code host supports squash merges, the source
+// must attempt a squash merge. Otherwise, it is expected to perform a regular
+// merge.
```

Comment:
I know this is copy/pasted from the interface, but we could be more specific here; eg:

```suggestion
// The squash parameter is ignored, as Bitbucket Server does not support
// squash merges.
```